src/modules/services/rabbitmq.nix

```diff
@@ -143,6 +143,22 @@ in
     env.RABBITMQ_ENABLED_PLUGINS_FILE = plugin_file;
     env.RABBITMQ_NODENAME = "rabbit@localhost";
 
-    processes.rabbitmq.exec = "${cfg.package}/bin/rabbitmq-server";
+    processes.rabbitmq = {
+      exec = "${cfg.package}/bin/rabbitmq-server";
+
+      process-compose = {
+        readiness_probe = {
+          exec.command = "${cfg.package}/bin/rabbitmq-diagnostics -q ping";
+          initial_delay_seconds = 10;
```

Comment:
This seems a tad bit too high for me.
It should be fine to have the check fail a few times in case the process starts up faster than 10 seconds.